src/test/java/eu/europeana/iiif/service/ManifestServiceTest.java

```diff
@@ -122,30 +127,48 @@ public void setupApiStub() {
                         .withHeader(CONTENT_TYPE, APPLICATION_JSON)
                         .withBody(ExampleData.EXAMPLE_RECORD_CHILD_RESPONSE)));
 
-        // Full Text API, return 404 for all unknown annotation pages
-        stubFor(head(urlPathMatching("/presentation/.*/.*/annopage/.*"))
-                .willReturn(aResponse()
-                        .withStatus(404)
-                        .withHeader(CONTENT_LENGTH, "0")));
-
-        // Full Text API, return 200 for proper HEAD request
-        stubFor(head(urlEqualTo(PRESENTATION + ExampleData.EXAMPLE_FULLTEXT_ID + ANNOPAGE + ExampleData.EXAMPLE_FULLTEXT_PAGENR))
-                .willReturn(aResponse()
-                        .withStatus(200)
-                        .withHeader(CONTENT_LENGTH, "0")));
 
-        // Full Text API, simulate server error
-        stubFor(head(urlEqualTo(PRESENTATION + EXAMPLE_ERROR_ID + ANNOPAGE + ExampleData.EXAMPLE_FULLTEXT_PAGENR))
-                .willReturn(aResponse()
-                        .withStatus(500)
-                        .withHeader(CONTENT_LENGTH, "0")));
 
-        // Full Text API, simulate (timeout?) exception
-        stubFor(head(urlEqualTo(PRESENTATION + EXAMPLE_TIMEOUT_ID + ANNOPAGE + ExampleData.EXAMPLE_FULLTEXT_PAGENR))
-                .willReturn(aResponse()
-                        .withFixedDelay(30000)
-                        .withStatus(200)
-                        .withHeader(CONTENT_LENGTH, "0")));
+        // Full Text API, return 200 for proper Summary request
+        stubFor(get(urlEqualTo(PRESENTATION + EXAMPLE_FULLTEXT_ID + ANNOPAGE))
+                        .willReturn(aResponse()
+                                            .withStatus(200)
+                                            .withHeader(CONTENT_TYPE, APPLICATION_JSON)
+                                            .withBody(ExampleData.EXAMPLE_FULLTEXT_SUMMARY_RESPONSE)));
+
+        // Full Text API, return 404 for unknown Summary request
+        stubFor(get(urlEqualTo(PRESENTATION + TEST_BLA + ANNOPAGE))
+                        .willReturn(aResponse()
+                                            .withStatus(404)
+                                            .withBody("{\"error\": \"Not Found\"}")));
+
+        // Full Text API, simulate server error for Summary request
+        stubFor(get(urlEqualTo(PRESENTATION + EXAMPLE_ERROR_ID + ANNOPAGE))
+                        .willReturn(aResponse()
+                                            .withHeader(CONTENT_TYPE, APPLICATION_JSON)
+                                            .withBody("{\"error\": \"Server error\"}")));
+
+        // Full Text API, simulate (timeout?) exception for Summary request
+        stubFor(get(urlEqualTo(PRESENTATION + EXAMPLE_TIMEOUT_ID + ANNOPAGE))
+                        .willReturn(aResponse()
+                                            .withFixedDelay(30000)
```

Comment:
Nice implementation! Can we make the delay shorter so we can run unit tests faster?

Reply:
I suppose so, I'll try with 5000